lumen/routes/web.php

```diff
@@ -17,12 +17,10 @@
     return $router->app->version();
 });
 
-$router->get('/events', function (MongoClient $client) {
-    $eventStorage = $client->test->events;
-    $cursor = $eventStorage->find();
-    $items = $cursor->toArray();
-    
-    return response()->json($items);
+$router->get('/events', function () {
+    $events = Event::paginate(10);
```

Comment:
Mongo Eloquent is cool, but you can also use https://laravel.com/api/5.0/Illuminate/Pagination/LengthAwarePaginator.html and calculate `last_page` as well as  the `offset` and `skip` for the mongo find options.

Reply:
Thanks! I'll take a look at the documentation, there's so much that I don't know about `Illuminate`